sql/synth_recipes.sql

```diff
@@ -1335,7 +1335,7 @@ INSERT INTO `synth_recipes` VALUES (1270,1,0,0,0,64,0,0,0,0,0,4098,4240,2149,214
 INSERT INTO `synth_recipes` VALUES (1271,1,2026,0,78,0,0,0,0,0,0,4098,4240,880,880,880,2130,2147,2147,2147,0,2293,2293,2293,2293,18,24,30,36);
 INSERT INTO `synth_recipes` VALUES (1272,1,0,0,78,0,0,0,0,0,0,4098,4240,880,2147,0,0,0,0,0,0,2293,2293,2293,2293,6,8,10,12);
 INSERT INTO `synth_recipes` VALUES (1273,1,0,77,0,0,0,0,0,0,0,4098,4240,648,937,2290,4159,0,0,0,0,2294,2294,2294,2294,6,8,10,12);
-INSERT INTO `synth_recipes` VALUES (1274,1,2010,0,0,41,0,0,0,0,0,4100,4242,2128,2295,2295,2295,0,0,0,0,2296,2296,2296,2296,3,3,3,3);
+INSERT INTO `synth_recipes` VALUES (1274,1,2010,0,0,41,0,0,0,0,0,4100,4242,2128,2295,2295,2295,2295,2295,2295,0,2296,2296,2296,2296,3,3,3,3);
```

Comment:
according to the link you provided, this is not correct

synth requires 3x grass not 6